src/app/components/MediaLoader/utils/buildSetting.test.ts

```diff
@@ -20,11 +20,11 @@ const baseSettings = {
   pageType: 'article' as PageTypes,
   counterName: 'live_coverage.testID.page',
   isAmp: false,
-  lang: 'es',
-  service: 'mundo' as Services,
+  lang: 'sr-latn',
+  service: 'serbian' as Services,
   statsDestination: 'WS_NEWS_LANGUAGES',
-  producer: 'MUNDO',
-  pathname: '/mundo/av-embeds/srbija-68707945',
+  producer: 'SERBIAN',
+  id: 'serbian/cyr/srbija-68707945',
```

Comment:
Let's have the variant match the lang 
```suggestion
  id: 'serbian/lat/srbija-68707945',
```